server/src/test/util/my-repos-test.ts

```diff
@@ -0,0 +1,29 @@
+import test from 'ava';
+
+import {getMyRepos} from '../../util/my-repos';
+import {startTestReplayServer} from '../replay-server';
+
+test.beforeEach(async (t) => {
+  const {server, client} = await startTestReplayServer(t);
+  t.context.server = server;
```

Comment:
Use the type safe ava context setup from that other recent PR?

Reply:
Done.